docs/components/MessagingThreadHistoryView.jsx

```diff
@@ -0,0 +1,125 @@
+import * as React from "react";
+import moment from "moment";
+
+import Example, { CodeSample, ExampleCode } from "./Example";
+import PropDocumentation from "./PropDocumentation";
+import View from "./View";
+import { MessagingThreadHistory, FlexBox, ItemAlign, Button, MessagingBubble } from "src";
+
+import "./MessagingThreadHistoryView.less";
+
+const cssClass = {
+  CONFIG_CONTAINER: "MessagingThreadHistoryView--configContainer",
+  CONFIG_OPTIONS: "MessagingThreadHistoryView--configOptions",
+  CONFIG: "MessagingThreadHistoryView--config",
+  CONFIG_TOGGLE: "MessagingThreadHistoryView--configToggle",
+  CONTAINER: "MessagingThreadHistoryView",
+  INTRO: "MessagingThreadHistoryView--intro",
+  PROPS: "MessagingThreadHistoryView--props",
+};
+
+// Start with an older time so we can better display the day separators.
+// We display any dates in the future as "today".
```

Comment:
Meant to address this at some point: Do you think that's undesirable behavior?

Similar question with regard to years, as the current dividers don't really display them. e.g. 5/8/2019 and 5/8/2020 are displayed identically.

Reply:
Hmm yeah good questions. In theory, we should never end up with times in the future, right? In practice it's the classic  "wtf how do timezones work??". But I think for now assuming we won't have times in the future seems okay, we can always revisit this later if need be.

I also think including years doesn't seem super important. I think the plan is to clear out old messages after a school year, right? So we probably won't end up with messages that are the same date in different years.

Reply:
Sounds good to me! 👍